package.json

```diff
@@ -115,16 +120,10 @@
     "terser": "^4.0.0",
     "uglifycss": "0.0.26",
     "webpack": "^4.35.3",
-    "webpack-cli": "^3.3.5",
-    "webpack-livereload-plugin": "^2.2.0"
+    "webpack-cli": "^3.3.5"
   },
   "config": {
-    "vendor_modules": [
-      "react",
-      "react-dom"
-    ],
     "bundles": {
-      "vendor": "build/vendor.js",
       "main": {
         "out": "build/main.js",
         "entry": "src/index.js"
```

Comment:
Just cleaning up the package json here.  There were a lot of unused fields.  Also adding class properties so we don't have to "bind" all the time.